handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -165,6 +166,12 @@ public void testSupportedSignatureAlgorithms(SSLEngineTestParam param) throws Ex
         super.testSupportedSignatureAlgorithms(param);
     }
 
+    @Override
+    protected void additionalPeerAssertions(SSLSession sslSession, boolean mutualAuth) {
+        assumeTrue(sslSession instanceof OpenSslSession);
```

Comment:
I think this should be an if-clause. Otherwise, we're aborting the whole test here, when other aspects of it might be relevant to check.

Reply:
I figured it might be a bug if the `OpenSslEngineTest` does not receive an `OpenSslSession` - happy to make the change though :)

Reply:
We have interoperability tests that use a mix.

Reply:
Gotcha, will fix.